src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,69 @@
+body {
+  margin: 0;
+  min-height: 100vh;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  width: 500px;
+  height: 500px;
+  background-color: aqua;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+
+  &__hands {
+    width: 400px;
+    height: 400px;
+    position: relative;
+
+    &-minutes {
+      position: absolute;
+      left: 50%;
+      top: 30px;
+      width: 5px;
+      height: 170px;
```

Comment:
I don't understand where do these numbers come from
Are they somehow related to the size of the watch itself (500px)

Reply:
Seconds hands are smaller in height with 30px than the Minutes hands.

But, I've changed them to make more sense.